### PR TITLE
[smartbroker] Adds the broker "smartbroker"

### LINKED ISSUE #152: Wertpapierabrechnungen smartbroker
<h2>Description</h2>
<blockquote>
<p>Wertpapierabrechnungen von smartbroker werden nicht erkannt</p>
</blockquote>
<h2>Document</h2>
<details><summary>K 2020-06-24.pdf</summary><p>
<pre><code class="language-json">[
  [
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;BNP Paribas S.A. Niederlassung Deutschland &gt; Standort: München &gt; Landsberger Straße 300 &gt; 80687 München &gt; Sitz: Nürnberg HRB Nürnberg 31129 &gt; USt.-Ident-Nr.: DE&quot;,
    &quot;191528929 &gt; Sitz der Hauptniederlassung der BNP Paribas S.A.: 16, boulevard des Italiens, 75009 Paris, Frankreich, Registergericht: R.C.S. Paris 662 042 449 &gt; Président du&quot;,
    &quot;Conseil d‘Administration: Jean Lemierre, Directeur Général: Jean-Laurent Bonnafé&quot;,
    &quot;Depot-Nr.&quot;,
    &quot;&quot;,
    &quot;Abrechnungs-Nr.&quot;,
    &quot;23411043 / 24.06.2020&quot;,
    &quot;&quot;,
    &quot;3.19/ABREABHNHANDKFDI/GAAASAFG/003808/240620/210727&quot;,
    &quot;München, 24.06.2020&quot;,
    &quot;BELEGDRUCK=J&quot;,
    &quot;ORIGINAL=1&quot;,
    &quot;FAXVERSAND=N&quot;,
    &quot;EMAILVERSAND=N&quot;,
    &quot;DEPOTNUMMER=5130560005&quot;,
    &quot;DEPOTUNTERBEZEICHNUNG=&quot;,
    &quot;VERSANDARTENSCHLUESSEL=DRUCK&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;ADRESSZEILE5=&quot;,
    &quot;ADRESSZEILE6=&quot;,
    &quot;BELEGNUMMER=3808&quot;,
    &quot;SEITENNUMMER=1&quot;,
    &quot;STEUERERSTATTUNG=N&quot;,
    &quot;Wertpapierabrechnung&quot;,
    &quot;Kauf&quot;,
    &quot;Kommissionsgeschäft&quot;,
    &quot;Gattungsbezeichnung&quot;,
    &quot;Abbott Laboratories Registered Shares o.N.&quot;,
    &quot;ISIN&quot;,
    &quot;US0028241000&quot;,
    &quot;Nominal&quot;,
    &quot;STK 14,000&quot;,
    &quot;Kurs&quot;,
    &quot;EUR 77,8600&quot;,
    &quot;Dieser Beleg wird maschinell erstellt und daher nicht unterschrieben.&quot;,
    &quot;Wir haben für Sie gekauft&quot;,
    &quot;Handelstag&quot;,
    &quot;24.06.2020&quot;,
    &quot;Handelszeit&quot;,
    &quot;17:33*&quot;,
    &quot;Börse&quot;,
    &quot;Gettex/GDM&quot;,
    &quot;Verwahrart&quot;,
    &quot;Girosammelverwahrung&quot;,
    &quot;Kurswert&quot;,
    &quot;EUR&quot;,
    &quot;1.090,04&quot;,
    &quot;-&quot;,
    &quot;Wert&quot;,
    &quot;26.06.2020&quot;,
    &quot;Konto-Nr.&quot;,
    &quot;&quot;,
    &quot;Betrag zu Ihren Lasten&quot;,
    &quot;EUR&quot;,
    &quot;1.090,04&quot;,
    &quot;Wir werden in Ihrem Depot wie angegeben buchen.&quot;,
    &quot;* Die angegebene Zeit entspricht den Angaben der Handelspartner gemäß der lokalen Zeit in Deutschland&quot;,
    &quot;(Mitteleuropäische Zeit bzw. Mitteleuropäische Sommerzeit).&quot;
  ]
]

</code></pre>
</p></details>
<hr />
<p>Issue submitted with https://import.tresor.one</p>

### LINKED ISSUE #153: Smartbroker Beispiel Dividendengutschrift Aktie
<h2>Description</h2>
<blockquote>
<p>Beispiel für eine Ertragsabrechnung Aktie mit Steuereinbehalt</p>
</blockquote>
<h2>Document</h2>
<details><summary>Dividenden_PEPSICO_INC______DL-,0166_vom_29_09_2020.pdf</summary><p>
<pre><code class="language-json">[
  [
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;BNP Paribas S.A. Niederlassung Deutschland &gt; Standort: München &gt; Landsberger Straße 300 &gt; 80687 München &gt; Sitz: Nürnberg HRB Nürnberg 31129 &gt; USt.-Ident-Nr.: DE&quot;,
    &quot;191528929 &gt; Sitz der Hauptniederlassung der BNP Paribas S.A.: 16, boulevard des Italiens, 75009 Paris, Frankreich, Registergericht: R.C.S. Paris 662 042 449 &gt; Président du&quot;,
    &quot;Conseil d‘Administration: Jean Lemierre, Directeur Général: Jean-Laurent Bonnafé&quot;,
    &quot;Depot-Nr.&quot;,
    &quot;&quot;,
    &quot;Abrechnungs-Nr.&quot;,
    &quot;24555982 / 29.09.2020&quot;,
    &quot;Depotinhaber&quot;,
    &quot;3.17/ABREEREIERTR/GDBERTRG/005023/300920/000715&quot;,
    &quot;München, 29.09.2020&quot;,
    &quot;BELEGDRUCK=J&quot;,
    &quot;ORIGINAL=1&quot;,
    &quot;FAXVERSAND=N&quot;,
    &quot;EMAILVERSAND=N&quot;,
    &quot;&quot;,
    &quot;DEPOTUNTERBEZEICHNUNG=&quot;,
    &quot;VERSANDARTENSCHLUESSEL=DRUCK&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;ADRESSZEILE5=&quot;,
    &quot;ADRESSZEILE6=&quot;,
    &quot;BELEGNUMMER=5023&quot;,
    &quot;SEITENNUMMER=1&quot;,
    &quot;STEUERERSTATTUNG=N&quot;,
    &quot;Dieser Beleg wird maschinell erstellt und daher nicht unterschrieben.&quot;,
    &quot;Erträgnisgutschrift aus Wertpapieren&quot;,
    &quot;Dividendengutschrift&quot;,
    &quot;Gattungsbezeichnung&quot;,
    &quot;PepsiCo Inc. Registered Shares DL -,0166&quot;,
    &quot;ISIN&quot;,
    &quot;US7134481081&quot;,
    &quot;Nominal&quot;,
    &quot;STK 9,000&quot;,
    &quot;Ex-Tag&quot;,
    &quot;03.09.2020&quot;,
    &quot;Zahltag&quot;,
    &quot;30.09.2020&quot;,
    &quot;Dividenden-Betrag pro Stück&quot;,
    &quot;USD 1,022500&quot;,
    &quot;Dividende für&quot;,
    &quot;01.01.2020 - 31.12.2020&quot;,
    &quot;Wert&quot;,
    &quot;30.09.2020&quot;,
    &quot;Konto-Nr.&quot;,
    &quot;&quot;,
    &quot;Devisenkurs&quot;,
    &quot;EUR/USD 1,1679&quot;,
    &quot;Betrag zu Ihren Gunsten&quot;,
    &quot;EUR&quot;,
    &quot;5,81&quot;,
    &quot;Hinweise zur steuerlichen Verrechnung:&quot;,
    &quot;vorher&quot;,
    &quot;aktuell&quot;,
    &quot;ausländische Dividende&quot;,
    &quot;EUR&quot;,
    &quot;7,88&quot;,
    &quot;US-Quellensteuer 15%&quot;,
    &quot;USD&quot;,
    &quot;1,38&quot;,
    &quot;davon anrechenbare US-Quellensteuer 15%&quot;,
    &quot;EUR&quot;,
    &quot;1,18&quot;,
    &quot;Aktienverlusttopf&quot;,
    &quot;EUR&quot;,
    &quot;0,00&quot;,
    &quot;0,00&quot;,
    &quot;allgemeiner Verlusttopf&quot;,
    &quot;EUR&quot;,
    &quot;0,00&quot;,
    &quot;0,00&quot;,
    &quot;Quellensteuertopf&quot;,
    &quot;EUR&quot;,
    &quot;0,00&quot;,
    &quot;0,00&quot;,
    &quot;zu versteuern&quot;,
    &quot;EUR&quot;,
    &quot;3,16&quot;,
    &quot;einbehaltene Kapitalertragsteuer&quot;,
    &quot;EUR&quot;,
    &quot;0,77&quot;,
    &quot;einbehaltener Solidaritätszuschlag&quot;,
    &quot;EUR&quot;,
    &quot;0,05&quot;,
    &quot;einbehaltene Kirchensteuer&quot;,
    &quot;EUR&quot;,
    &quot;0,07&quot;,
    &quot;im laufenden Jahr einbehaltene Kapitalertragsteuer&quot;,
    &quot;EUR&quot;,
    &quot;&quot;,
    &quot;im laufenden Jahr einbehaltener Solidaritätszuschlag&quot;,
    &quot;EUR&quot;,
    &quot;&quot;,
    &quot;im laufenden Jahr einbehaltene Kirchensteuer&quot;,
    &quot;EUR&quot;,
    &quot;&quot;,
    &quot;Kapitalertragsteuer, Solidaritätszuschlag und ggf. Kirchensteuer nach gemeldetem Kirchensteuersatz verrechnet mit dem&quot;,
    &quot;Finanzamt Frankfurt/M. V-Höchst, Steuernummer 047/220/15370.&quot;,
    &quot;Steuerlicher Stichtag: 30.09.2020&quot;,
    &quot;Jahressteuerbescheinigung folgt&quot;,
    &quot;Verwahrart: Girosammelverwahrung&quot;
  ]
]

</code></pre>
</p></details>
<hr />
<p>Issue submitted with https://import.tresor.one</p>

### LINKED ISSUE #154: Smartbroker Beispiel Ertragsausschüttung ETF
<h2>Description</h2>
<blockquote>
<p>Beispiel für eine Ertragsabrechnung ETF mit Steuereinbehalt</p>
</blockquote>
<h2>Document</h2>
<details><summary>Ausschuettung_VANG_USD_EM_G_BD_UETF_DLD_vom_09_10_2020.pdf</summary><p>
<pre><code class="language-json">[
  [
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;BNP Paribas S.A. Niederlassung Deutschland &gt; Standort: München &gt; Landsberger Straße 300 &gt; 80687 München &gt; Sitz: Nürnberg HRB Nürnberg 31129 &gt; USt.-Ident-Nr.: DE&quot;,
    &quot;191528929 &gt; Sitz der Hauptniederlassung der BNP Paribas S.A.: 16, boulevard des Italiens, 75009 Paris, Frankreich, Registergericht: R.C.S. Paris 662 042 449 &gt; Président du&quot;,
    &quot;Conseil d‘Administration: Jean Lemierre, Directeur Général: Jean-Laurent Bonnafé&quot;,
    &quot;Depot-Nr.&quot;,
    &quot;&quot;,
    &quot;Abrechnungs-Nr.&quot;,
    &quot;21541229 / 09.10.2020&quot;,
    &quot;Depotinhaber&quot;,
    &quot;3.17/ABREEREIERTR/GDBERTRG/008526/091020/015334&quot;,
    &quot;München, 09.10.2020&quot;,
    &quot;BELEGDRUCK=J&quot;,
    &quot;ORIGINAL=1&quot;,
    &quot;FAXVERSAND=N&quot;,
    &quot;EMAILVERSAND=N&quot;,
    &quot;&quot;,
    &quot;DEPOTUNTERBEZEICHNUNG=&quot;,
    &quot;VERSANDARTENSCHLUESSEL=DRUCK&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;ADRESSZEILE5=&quot;,
    &quot;ADRESSZEILE6=&quot;,
    &quot;BELEGNUMMER=8526&quot;,
    &quot;SEITENNUMMER=1&quot;,
    &quot;STEUERERSTATTUNG=N&quot;,
    &quot;Erträgnisgutschrift aus Wertpapieren&quot;,
    &quot;Erträgnisgutschrift&quot;,
    &quot;Gattungsbezeichnung&quot;,
    &quot;Vang.USD Em.Mkts Gov.Bd U.ETF Registered Shares USD Dis.oN&quot;,
    &quot;ISIN&quot;,
    &quot;IE00BZ163L38&quot;,
    &quot;Nominal&quot;,
    &quot;STK 445,000&quot;,
    &quot;Ex-Tag&quot;,
    &quot;24.09.2020&quot;,
    &quot;Zahltag&quot;,
    &quot;07.10.2020&quot;,
    &quot;Ausschüttungsbetrag pro Stück&quot;,
    &quot;USD 0,193781&quot;,
    &quot;Ausschüttung für&quot;,
    &quot;01.07.2020 - 30.06.2021&quot;,
    &quot;Es folgt Seite&quot;,
    &quot;2&quot;,
    &quot;Wert&quot;,
    &quot;07.10.2020&quot;,
    &quot;Konto-Nr.&quot;,
    &quot;&quot;,
    &quot;Devisenkurs&quot;,
    &quot;EUR/USD 1,1804&quot;,
    &quot;Betrag zu Ihren Gunsten&quot;,
    &quot;EUR&quot;,
    &quot;52,60&quot;,
    &quot;Ertrag für 2020/21&quot;,
    &quot;USD&quot;,
    &quot;86,23&quot;,
    &quot;Hinweise zur steuerlichen Verrechnung:&quot;,
    &quot;vorher&quot;,
    &quot;aktuell&quot;,
    &quot;Steuerpflichtiger Ausschüttungsbetrag&quot;,
    &quot;EUR&quot;,
    &quot;73,05&quot;,
    &quot;Aktienverlusttopf&quot;,
    &quot;EUR&quot;,
    &quot;0,00&quot;,
    &quot;0,00&quot;,
    &quot;allgemeiner Verlusttopf&quot;,
    &quot;EUR&quot;,
    &quot;0,00&quot;,
    &quot;0,00&quot;,
    &quot;Quellensteuertopf&quot;,
    &quot;EUR&quot;,
    &quot;0,00&quot;,
    &quot;0,00&quot;,
    &quot;zu versteuern&quot;,
    &quot;EUR&quot;,
    &quot;73,05&quot;,
    &quot;einbehaltene Kapitalertragsteuer&quot;,
    &quot;EUR&quot;,
    &quot;17,86&quot;,
    &quot;einbehaltener Solidaritätszuschlag&quot;,
    &quot;EUR&quot;,
    &quot;0,98&quot;,
    &quot;einbehaltene Kirchensteuer&quot;,
    &quot;EUR&quot;,
    &quot;1,61&quot;,
    &quot;im laufenden Jahr einbehaltene Kapitalertragsteuer&quot;,
    &quot;EUR&quot;,
    &quot;&quot;,
    &quot;im laufenden Jahr einbehaltener Solidaritätszuschlag&quot;,
    &quot;EUR&quot;,
    &quot;&quot;,
    &quot;im laufenden Jahr einbehaltene Kirchensteuer&quot;,
    &quot;EUR&quot;,
    &quot;&quot;,
    &quot;Kapitalertragsteuer, Solidaritätszuschlag und ggf. Kirchensteuer nach gemeldetem Kirchensteuersatz verrechnet mit dem&quot;,
    &quot;Finanzamt Frankfurt/M. V-Höchst, Steuernummer 047/220/15370.&quot;,
    &quot;Steuerlicher Stichtag: 07.10.2020&quot;,
    &quot;Jahressteuerbescheinigung folgt&quot;
  ],
  [
    &quot;BNP Paribas S.A. Niederlassung Deutschland &gt; Standort: München &gt; Landsberger Straße 300 &gt; 80687 München &gt; Sitz: Nürnberg HRB Nürnberg 31129 &gt; USt.-Ident-Nr.: DE&quot;,
    &quot;191528929 &gt; Sitz der Hauptniederlassung der BNP Paribas S.A.: 16, boulevard des Italiens, 75009 Paris, Frankreich, Registergericht: R.C.S. Paris 662 042 449 &gt; Président du&quot;,
    &quot;Conseil d‘Administration: Jean Lemierre, Directeur Général: Jean-Laurent Bonnafé&quot;,
    &quot;Depot-Nr.&quot;,
    &quot;Abrechnungs-Nr.&quot;,
    &quot;Seite-Nr.&quot;,
    &quot;&quot;,
    &quot;21541229&quot;,
    &quot;2&quot;,
    &quot;3.17/ABREEREIERTR/GDBERTRG/008526/091020/015334&quot;,
    &quot;BELEGDRUCK=J&quot;,
    &quot;ORIGINAL=1&quot;,
    &quot;FAXVERSAND=N&quot;,
    &quot;EMAILVERSAND=N&quot;,
    &quot;DEPOTNUMMER=5130560005&quot;,
    &quot;DEPOTUNTERBEZEICHNUNG=&quot;,
    &quot;VERSANDARTENSCHLUESSEL=DRUCK&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;&quot;,
    &quot;ADRESSZEILE5=&quot;,
    &quot;ADRESSZEILE6=&quot;,
    &quot;BELEGNUMMER=8526&quot;,
    &quot;SEITENNUMMER=2&quot;,
    &quot;STEUERERSTATTUNG=N&quot;,
    &quot;Dieser Beleg wird maschinell erstellt und daher nicht unterschrieben.&quot;,
    &quot;Fondsart:&quot;,
    &quot;Sonstiger Investmentfonds&quot;,
    &quot;Angewendete Teilfreistellungsquote: 0%&quot;,
    &quot;Im Steuerabzugsverfahren werden generell - auch bei betrieblichen Anlegern - die Teilfreistellungsquoten (TFQ) für&quot;,
    &quot;Privatanleger herangezogen.&quot;,
    &quot;Verwahrart: Wertpapierrechnung&quot;,
    &quot;Lagerland: Großbritannien&quot;
  ]
]

</code></pre>
</p></details>
<hr />
<p>Issue submitted with https://import.tresor.one</p>

### LINKED ISSUE #195: [Smartbroker] Quellensteuer fehlt bei Dividendenimport
<h2>Description</h2>
<blockquote>
<p>Die Steuern werden beim Import nur teilweise berücksichtigt. Ich erwarte beim Import eine Summe von 5,14€ (statt aktuell 4,37€) und Steuern in Höhe von 1,34€ (die aktuell importierten 0,57€ plus die fehlende Quellensteuer in Höhe von 0,77€)</p>
</blockquote>
<h2>Document</h2>
<details><summary>Dividenden_LTC_PTIES_INC______DL-,01_vom_29_10_2020.pdf</summary><p>
<pre><code class="language-json">[
  [
    &quot;Herr&quot;,
    &quot;XXX&quot;,
    &quot;XXX&quot;,
    &quot;XXX&quot;,
    &quot;XXX&quot;,
    &quot;BNP Paribas S.A. Niederlassung Deutschland &gt; Standort: München &gt; Landsberger Straße 300 &gt; 80687 München &gt; Sitz: Nürnberg HRB Nürnberg 31129 &gt; USt.-Ident-Nr.: DE&quot;,
    &quot;191528929 &gt; Sitz der Hauptniederlassung der BNP Paribas S.A.: 16, boulevard des Italiens, 75009 Paris, Frankreich, Registergericht: R.C.S. Paris 662 042 449 &gt; Président du&quot;,
    &quot;Conseil d‘Administration: Jean Lemierre, Directeur Général: Jean-Laurent Bonnafé&quot;,
    &quot;Depot-Nr.&quot;,
    &quot;XXX&quot;,
    &quot;Abrechnungs-Nr.&quot;,
    &quot;69892042 / 29.10.2020&quot;,
    &quot;Depotinhaber&quot;,
    &quot;3.17/ABREEREIERTR/GDBERTRG/003161/301020/000715&quot;,
    &quot;München, 29.10.2020&quot;,
    &quot;BELEGDRUCK=J&quot;,
    &quot;ORIGINAL=1&quot;,
    &quot;FAXVERSAND=N&quot;,
    &quot;EMAILVERSAND=N&quot;,
    &quot;DEPOTNUMMER=XXX&quot;,
    &quot;DEPOTUNTERBEZEICHNUNG=&quot;,
    &quot;VERSANDARTENSCHLUESSEL=DRUCK&quot;,
    &quot;ADRESSZEILE1=Herr&quot;,
    &quot;ADRESSZEILE2=XXX&quot;,
    &quot;ADRESSZEILE3=XXX&quot;,
    &quot;ADRESSZEILE4=XXX&quot;,
    &quot;ADRESSZEILE5=&quot;,
    &quot;ADRESSZEILE6=&quot;,
    &quot;BELEGNUMMER=3161&quot;,
    &quot;SEITENNUMMER=1&quot;,
    &quot;STEUERERSTATTUNG=N&quot;,
    &quot;Dieser Beleg wird maschinell erstellt und daher nicht unterschrieben.&quot;,
    &quot;Erträgnisgutschrift aus Wertpapieren&quot;,
    &quot;Dividendengutschrift&quot;,
    &quot;Gattungsbezeichnung&quot;,
    &quot;LTC Properties Inc. Registered Shares DL -,01&quot;,
    &quot;ISIN&quot;,
    &quot;US5021751020&quot;,
    &quot;Nominal&quot;,
    &quot;STK 32,000&quot;,
    &quot;Ex-Tag&quot;,
    &quot;21.10.2020&quot;,
    &quot;Zahltag&quot;,
    &quot;30.10.2020&quot;,
    &quot;Dividenden-Betrag pro Stück&quot;,
    &quot;USD 0,190000&quot;,
    &quot;Dividende für&quot;,
    &quot;01.07.2020 - 30.06.2021&quot;,
    &quot;Wert&quot;,
    &quot;30.10.2020&quot;,
    &quot;Konto-Nr.&quot;,
    &quot;XXX&quot;,
    &quot;Devisenkurs&quot;,
    &quot;EUR/USD 1,1821&quot;,
    &quot;Betrag zu Ihren Gunsten&quot;,
    &quot;EUR&quot;,
    &quot;3,80&quot;,
    &quot;Hinweise zur steuerlichen Verrechnung:&quot;,
    &quot;vorher&quot;,
    &quot;aktuell&quot;,
    &quot;Steuerpflichtiger Ausschüttungsbetrag&quot;,
    &quot;EUR&quot;,
    &quot;5,14&quot;,
    &quot;US-Quellensteuer 15%&quot;,
    &quot;USD&quot;,
    &quot;0,91&quot;,
    &quot;davon anrechenbare US-Quellensteuer 15%&quot;,
    &quot;EUR&quot;,
    &quot;0,77&quot;,
    &quot;Aktienverlusttopf&quot;,
    &quot;EUR&quot;,
    &quot;0,00&quot;,
    &quot;0,00&quot;,
    &quot;allgemeiner Verlusttopf&quot;,
    &quot;EUR&quot;,
    &quot;0,00&quot;,
    &quot;0,00&quot;,
    &quot;Quellensteuertopf&quot;,
    &quot;EUR&quot;,
    &quot;0,00&quot;,
    &quot;0,00&quot;,
    &quot;zu versteuern&quot;,
    &quot;EUR&quot;,
    &quot;2,06&quot;,
    &quot;einbehaltene Kapitalertragsteuer&quot;,
    &quot;EUR&quot;,
    &quot;0,51&quot;,
    &quot;einbehaltener Solidaritätszuschlag&quot;,
    &quot;EUR&quot;,
    &quot;0,02&quot;,
    &quot;einbehaltene Kirchensteuer&quot;,
    &quot;EUR&quot;,
    &quot;0,04&quot;,
    &quot;im laufenden Jahr einbehaltene Kapitalertragsteuer&quot;,
    &quot;EUR&quot;,
    &quot;6,54&quot;,
    &quot;im laufenden Jahr einbehaltener Solidaritätszuschlag&quot;,
    &quot;EUR&quot;,
    &quot;0,35&quot;,
    &quot;im laufenden Jahr einbehaltene Kirchensteuer&quot;,
    &quot;EUR&quot;,
    &quot;0,52&quot;,
    &quot;Kapitalertragsteuer, Solidaritätszuschlag und ggf. Kirchensteuer nach gemeldetem Kirchensteuersatz verrechnet mit dem&quot;,
    &quot;Finanzamt Frankfurt/M. V-Höchst, Steuernummer 047/220/15370.&quot;,
    &quot;Steuerlicher Stichtag: 30.10.2020&quot;,
    &quot;Jahressteuerbescheinigung folgt&quot;,
    &quot;Verwahrart: Wertpapierrechnung&quot;,
    &quot;Lagerland: USA&quot;
  ]
]

</code></pre>
</p></details>
<h2>Community profile</h2>
<p>https://community.tresor.one/u/97e9f35e</p>
<hr />
<p>Issue submitted with https://import.tresor.one</p>

### LINKED ISSUE #208: [Smartbroker] Steuern als Gebühren erkannt
## Document
<details><summary>Verkauf_Investmentfonds_VANG_FTSE_A_-WO_U_ETF_DLD_vom_05_11_2020.pdf</summary><p>

```json
[
  [
    "Herr",
    "XXX",
    "XXX",
    "XXX",
    "XXX",
    "BNP Paribas S.A. Niederlassung Deutschland > Standort: München > Landsberger Straße 300 > 80687 München > Sitz: Nürnberg HRB Nürnberg 31129 > USt.-Ident-Nr.: DE",
    "191528929 > Sitz der Hauptniederlassung der BNP Paribas S.A.: 16, boulevard des Italiens, 75009 Paris, Frankreich, Registergericht: R.C.S. Paris 662 042 449 > Président du",
    "Conseil d‘Administration: Jean Lemierre, Directeur Général: Jean-Laurent Bonnafé",
    "Depot-Nr.",
    "XXX",
    "Abrechnungs-Nr.",
    "XXX",
    "Depotinhaber",
    "3.19/ABREABHNHANDVFDI/GAAASAFG/003584/051120/110847",
    "München, 05.11.2020",
    "BELEGDRUCK=J",
    "ORIGINAL=1",
    "FAXVERSAND=N",
    "EMAILVERSAND=N",
    "DEPOTNUMMER=XXX",
    "DEPOTUNTERBEZEICHNUNG=",
    "VERSANDARTENSCHLUESSEL=DRUCK",
    "ADRESSZEILE1=Herr",
    "ADRESSZEILE2=XXX",
    "ADRESSZEILE3=XXX",
    "ADRESSZEILE4=XXX",
    "ADRESSZEILE5=",
    "ADRESSZEILE6=",
    "BELEGNUMMER=3584",
    "SEITENNUMMER=1",
    "STEUERERSTATTUNG=N",
    "Wertpapierabrechnung",
    "Verkauf",
    "Kommissionsgeschäft",
    "Gattungsbezeichnung",
    "Vanguard FTSE All-World U.ETF Registered Shares USD Dis.oN",
    "ISIN",
    "IE00B3RBWM25",
    "Nominal",
    "STK 26,000",
    "Kurs",
    "EUR 82,3000",
    "Es folgt Seite",
    "2",
    "Wir haben für Sie verkauft",
    "Handelstag",
    "05.11.2020",
    "Handelszeit",
    "09:48*",
    "Börse",
    "Gettex/GDM",
    "Verwahrart",
    "Wertpapierrechnung",
    "Lagerland",
    "Großbritannien",
    "Kurswert",
    "EUR",
    "2.139,80",
    "Kapitalertragsteuer",
    "EUR",
    "24,29",
    "-",
    "Solidaritätszuschlag",
    "EUR",
    "1,34",
    "-",
    "Kirchensteuer",
    "EUR",
    "1,94",
    "-",
    "Wert",
    "09.11.2020",
    "Konto-Nr.",
    "XXX",
    "Betrag zu Ihren Gunsten",
    "EUR",
    "2.112,23",
    "Hinweise zur steuerlichen Verrechnung:",
    "vorher",
    "aktuell",
    "Gesamtveräußerungsergebnis vor Anwendung einer TFQ",
    "EUR",
    "141,56",
    "Veräußerungsgewinn Fonds",
    "EUR",
    "99,09",
    "Aktienverlusttopf",
    "EUR",
    "0,00",
    "0,00",
    "allgemeiner Verlusttopf",
    "EUR",
    "0,00",
    "0,00",
    "Quellensteuertopf",
    "EUR",
    "0,00",
    "0,00",
    "zu versteuern",
    "EUR",
    "99,09",
    "im laufenden Jahr einbehaltene Kapitalertragsteuer",
    "EUR",
    "30,83",
    "im laufenden Jahr einbehaltener Solidaritätszuschlag",
    "EUR",
    "1,69",
    "im laufenden Jahr einbehaltene Kirchensteuer",
    "EUR",
    "2,46",
    "Kapitalertragsteuer, Solidaritätszuschlag und ggf. Kirchensteuer nach gemeldetem Kirchensteuersatz verrechnet mit dem",
    "Finanzamt Frankfurt/M. V-Höchst, Steuernummer 047/220/15370."
  ],
  [
    "BNP Paribas S.A. Niederlassung Deutschland > Standort: München > Landsberger Straße 300 > 80687 München > Sitz: Nürnberg HRB Nürnberg 31129 > USt.-Ident-Nr.: DE",
    "191528929 > Sitz der Hauptniederlassung der BNP Paribas S.A.: 16, boulevard des Italiens, 75009 Paris, Frankreich, Registergericht: R.C.S. Paris 662 042 449 > Président du",
    "Conseil d‘Administration: Jean Lemierre, Directeur Général: Jean-Laurent Bonnafé",
    "Depot-Nr.",
    "Abrechnungs-Nr.",
    "Seite-Nr.",
    "XXX",
    "37982092",
    "2",
    "3.19/ABREABHNHANDVFDI/GAAASAFG/003584/051120/110847",
    "BELEGDRUCK=J",
    "ORIGINAL=1",
    "FAXVERSAND=N",
    "EMAILVERSAND=N",
    "DEPOTNUMMER=XXX",
    "DEPOTUNTERBEZEICHNUNG=",
    "VERSANDARTENSCHLUESSEL=DRUCK",
    "ADRESSZEILE1=Herr",
    "ADRESSZEILE2=XXX",
    "ADRESSZEILE3=XXX",
    "ADRESSZEILE4=XXX",
    "ADRESSZEILE5=",
    "ADRESSZEILE6=",
    "BELEGNUMMER=3584",
    "SEITENNUMMER=2",
    "STEUERERSTATTUNG=N",
    "Dieser Beleg wird maschinell erstellt und daher nicht unterschrieben.",
    "Jahressteuerbescheinigung folgt",
    "Die Höhe des von der KVG festgelegten Ausgabeauf- bzw. Rücknahmeabschlages sowie die jährlich zu zahlende Ver-",
    "waltungsvergütung und die Gesamtkostenquote sind dem Verkaufsprospekt bzw. den Vertragsbedingungen zu entnehmen.",
    "Fondsart:",
    "Aktienfonds (§ 2 Abs. 6 InvStG)",
    "Angewendete Teilfreistellungsquote: 30%",
    "Im Steuerabzugsverfahren werden generell - auch bei betrieblichen Anlegern - die Teilfreistellungsquoten (TFQ) für",
    "Privatanleger herangezogen.",
    "Höhe der Teilfreistellung",
    "Privatanleger",
    "Betriebliche Anleger",
    "(natürliche Person)",
    "Körperschaft",
    "Aktienfonds",
    "30%",
    "60%",
    "80%",
    "Mischfonds",
    "15%",
    "30%",
    "40%",
    "Immobilienfonds",
    "60%",
    "60%",
    "60%",
    "Immobilienfonds mit",
    "überwiegend",
    "Auslandsimmobilien",
    "80%",
    "80%",
    "80%",
    "Sonstiger Investmentfonds",
    "0%",
    "0%",
    "0%",
    "* Die angegebene Zeit entspricht den Angaben der Handelspartner gemäß der lokalen Zeit in Deutschland (Mitteleuropäische Zeit bzw. Mitteleuropäische Sommerzeit).",
    "Widerrufsbelehrung für den Erwerb von Anteilen oder Aktien eines offenen Investmentvermögens: Ist der Käufer von Anteilen oder Aktien eines offenen Investmentvermögens",
    "durch mündliche Verhandlungen außerhalb der ständigen Geschäftsräume desjenigen, der die Anteile oder Aktien verkauft oder den Verkauf vermittelt hat, dazu bestimmt worden,",
    "eine auf den Kauf gerichtete Willenserklärung abzugeben, so ist er an diese Erklärung nur gebunden, wenn er sie nicht innerhalb einer Frist von zwei Wochen bei der",
    "Verwaltungsgesellschaft oder einem Repräsentanten im Sinne des § 319 KAGB schriftlich widerruft."
  ]
]

```

</p></details>

## Community profile

https://community.tresor.one/u/97e9f35e

---
Issue submitted with https://import.tresor.one

### PR DESCRIPTION
**Pull request by SirGibihm**
Added the broker "Smartbroker" which is very similar to the onvista broker since both are owned by the Bank BNP Paribas. Due to this fact I decided that smartbroker inherits from onvista instead of rewriting everything (of course I noticed this AFTER I allready wrote smartbroker :D ).  
It might be reasonable to create an abstract BNP-Paribas broker from which both onvista and smartbroker inherit instead of smartbroker inheriting from onvista directly. Will do this if another bnp-paribas broker comes along.   

Fix #195 
Fix #154 
Fix #153 
Fix #152 
Fix #208

### CODE REVIEW

File: src/brokers/smartbroker.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import { Big } from 'big.js';
+import { parseGermanNum, validateActivity } from '@/helper';
+import * as onvista from './onvista';
+
+export const smartbrokerIdentificationString = 'Landsberger Straße 300';
```
Comment (dredav) [inline src/brokers/smartbroker.js:5]: Did we need this `smartbrokerIdentificationString` or can we use `onvista. smartbrokerIdentificationString`?

File: src/brokers/smartbroker.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    amount: amount,
+    tax: tax,
+    fee: fee,
+  };
+  return validateActivity(activity);
+};
+
+export const parsePages = contents => {
+  // parse first page has activity data
+  const activities = [parseData(contents[0])];
```
Comment (dredav) [inline src/brokers/smartbroker.js:102]: I'm not sure about this. onvista documents contains all orders / activities from one day - based on the resemblance to onvista, I would prefer to check all pages like onvista.

**Review by dredav — COMMENTED** [review]
After a long time (sorry for that!) - I found time to review your `smartbroker` implementation. Looks good!

File: src/brokers/smartbroker.js (hunk 1 above)
Comment (SirGibihm) [reply src/brokers/smartbroker.js:5]: yes, we can. Fixed it and added further explanation in code

**Comment by SirGibihm** [comment]
I implemented all required changes and rebased everything to get rid of pesky merge conflicts by rebasing everything

**Review by dredav — APPROVED** [review]

**Pull request merged**